tests/third_party/k8s/framework.go

```diff
@@ -490,10 +489,6 @@ func (f *Framework) CreateNamespace(baseName string, labels map[string]string) (
 	// fail to create serviceAccount in it.
 	f.AddNamespacesToDelete(ns)
 
-	if err == nil && !f.SkipPrivilegedPSPBinding {
-		CreatePrivilegedPSPBinding(f.ClientSet, ns.Name)
-	}
```

Comment:
Pod Security Policies was removed from Kubernetes in v1.25, ref https://kubernetes.io/docs/concepts/security/pod-security-policy/